browser/mapping_test.go

```diff
@@ -343,6 +343,7 @@ type pageAPI interface {
 	SetViewportSize(viewportSize goja.Value)
 	Tap(selector string, opts goja.Value)
 	TextContent(selector string, opts goja.Value) string
+	throttleCPU(common.CPUProfile) error
```

Comment:
Should this be capitalized?

Reply:
Fixed in 51207e4c98548658bc1baa7d2f32dc5f79230586